IHM1_F20_sql

Jia-FU-Soline opened the pull request:
C'est juste pour montrer ma logique mais, mais j'ai pas des données encore du coup je sais pas si ça affichera bien. D'ailleurs, je trouve pas de Capacités dans les Tables.

Jia-FU-Soline committed:
IHM1_F20_sql

C'est juste pour montrer ma logique mais, mais j'ai pas des données encore du coup je sais pas si ça affichera bien. D'ailleurs, je trouve pas de Capacités dans les Tables.

fabien-ors commented:
Le dernier DER indique Cap pour le paramètre Capacité : https://drive.google.com/drive/folders/1tgpfaYN06f09ArDmwh60mpxytB7-4ZfZ
Concernant le manque de données. En attendant une BDD totalement convertie de v1 => v2 (Alexis), il est possible de se créer ses enregistrements dans les tables LastParameters et Layer avec INSERT. 
La première fois que la DialogCompute est ouverte pour un point donné, il n'y a pas de paramètres et on affiche les layers et les paramètres par défaut. Lorsque l'utilisateur valide la fenêtre, il faut insérer dans la BDD les paramètres (INSERT). Le prochain coup, il est donc possible de relire ces paramètres avec SELECT. Cela permet de tester sans attendre d'avoir une base convertie.

Jia-FU-Soline committed:
IHM1_F20_JF_sql

J'ai ajouté des codes pour UPDATE le bestParameters dans cette version. Mais pour le moment je sais pas si ca va marcher sans l'avoir testé avec la BDD.

Jia-FU-Soline committed:
IHM1_F20_JF_sql

Remove then insert

Jia-FU-Soline committed:
IHM1_F20_JF_sql

Jia-FU-Soline committed:
IHM1_F20_JF_sql

Jia-FU-Soline committed:
IHM1_F20_JF_sql

ca a marché. Je l'ai testé en mettant des chiffres dans les tableaux de .sqlite

Jia-FU-Soline committed:
IHM1_F20_JF_sql

Testé en mettant des chiffres dans les tableaux de sqlite
